test/completion/pep0484_typing.py

```diff
@@ -283,6 +283,18 @@ def testnewtype2(y):
     y
     #? []
     y.
+
+# The type of a NewType is equivalent to the type of its underlying type.
```

Comment:
(for lack of a better place to ask this) Is there a better place to put these tests? It's obviously really convenient to keep adding to these `pep0484_*.py` files, though they're not always the easiest to ensure isolation between cases for.

Is there somewhere else I should be considering when adding tests of this kind in future?

Reply:
No, at the moment there is no such thing. I feel like adding the tests there is fine. It hasn't really caused any issues in the past.